controllers/controllers.app.js

```diff
@@ -28,3 +29,12 @@ exports.getArticlesByArticleID = (req, res, next) => {
     })
     .catch(next);
 };
+
```

Comment:
Here you may need a checkIfArticleExists function to differentiate between a valid article with no comments (200), and a non existent article (404) . Both of these will return an empty array but need to be handled differently